pages/apim/3.x/installation-guide/rest-apis/installation-guide-rest-apis-configuration.adoc

```diff
@@ -283,6 +283,16 @@ user.creation.token.expire-after:: number (default: 86400) - Number of seconds b
 user.reference.secret:: 32 characters (default: s3cR3t4grAv1t33.1Ous3D4R3f3r3nc3) - CHANGE IT! secret used to generate a unique anonymous reference to a user.
 user.anonymize-on-delete:enabled:: boolean (default: false) - If true, the users firstname, lastname and email are anonymized when a user is deleted.
 
+== Gateway
+
+Gateway configuration. The field `unknown-expire-after` specify the visibility duration (in second) of a gateway with the state *unknown*. (default value : 7 days)
```

Comment:
The field `unknown-expire-after' specifies the duration of visibility (in seconds) of a gateway with the state *unknown* (default value: 7 days).